.travis.yml

```diff
@@ -49,7 +49,9 @@ script:
     - mypy src --ignore-missing-imports
     - if [[ "$FULL_INSTALL" == "1" ]]; then
         jupyter --version;
-        source docs/notebooks/run_notebooks.sh;
+        chmod +x docs/notebooks/run_notebooks.sh;
+        bash docs/notebooks/run_notebooks.sh;
```

Comment:
Is there a reason for generating the `run_selected_notebooks.sh` instead of executing the commands directly?

Reply:
No reason anymore with the current approach. I will change it so we don't need a temporary file. It didn't work with an earlier version.